tasks/main.yml

```diff
@@ -48,6 +48,34 @@
     (ansible_os_family != "Darwin" and nix_directory is changed) or
     (ansible_os_family == "Darwin" and not nix_directory_mac.stat.exists)
 
+- name: Create /nix bind mount
+  when:
+    - ansible_os_family != "Darwin"
+    - nix_bind_mount != ""
+  block:
+    - name: Check if bind mount target directory is present
+      ansible.builtin.stat:
+        path: "{{ nix_bind_mount }}"
+      register: nix_bind_mount_stat
+
+    - name: Create bind mount target directory
+      become: true
+      ansible.builtin.file:
+        path: "{{ nix_bind_mount }}"
+        owner: "{{ nix_user }}"
+        state: directory
+        mode: "0755"
+      when: not nix_bind_mount_stat.stat.exists  # noqa: no-handler
+
+    - name: "Bind mount /nix to {{ nix_bind_mount }}"
+      become: true
+      ansible.posix.mount:
+        path: "{{ nix_bind_mount }}"
+        src: "/nix"
```

Comment:
It should be the other way around.

Reply:
See https://github.com/AbletonAppDev/devtools/issues/5739.